### PR TITLE
docs: add cdk8s+ FAQ

### PR DESCRIPTION
**Pull request by Chriscbr**
*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: docs/plus/index.md [hunk 1]
```diff
@@ -24,6 +24,42 @@ any spec version higher or equal to `1.22.0`. **Non-stable** resources are not g
...
+
+If you are using stable APIs (those that are not in alpha or beta), manifests
+generated in `cdk8s-plus-XX` will work in Kubernetes versions `1.XX.0` and
+above. Unstable APIs (which are always labeled in cdk8s+ using a suffix, e.g.
+`IngressV1Beta1`) may work in newer versions of Kubernetes, but it is also
+possible they have been removed.
+
+The manifests generated by `cdk8s-plus-XX` may also work for older versions of
+Kubernetes, but you might encounter some unsupported spec properties or invalid
+manifests.
```
Comment (Chriscbr) [inline docs/plus/index.md:61]: This sentence is repeated from the section above, but I think that is OK. I'm mostly aiming for brevity and accuracy

File: docs/plus/index.md [hunk 2]
```diff
@@ -24,6 +24,42 @@ any spec version higher or equal to `1.22.0`. **Non-stable** resources are not g
...
 
+## FAQ
+
+### What's the difference between `cdk8s-plus-20`, `cdk8s-plus-21`, and `cdk8s-plus-22`?
+
+These are separately vended libraries that each target a different kubernetes
+version, marked by the `-XX` suffix. For example, `cdk8s-plus-22` targets
+kubernetes version `1.22.0`.
+
+We offer a dedicated version per Kubernetes version to allow users to match
```
Comment (iliapolo) [inline docs/plus/index.md:35]:
```suggestion
We offer a dedicated package per Kubernetes version to allow users to match
```

**Review by iliapolo — APPROVED** [review]
Cool. I agree with brevity - lets see if we need to expand this based on user feedback :)

File: docs/plus/index.md (hunk 1 above)
Comment (iliapolo) [reply docs/plus/index.md:61]: 👍

**Pull request merged**